### ISSUE TITLE
Validate a single view

### ISSUE THREAD

**Issue by pablobaldez** [ragunathjawahar/android-saripaar#79]
Hi 

there are a way to validate a single view in my form? 
My use case:
one form with N edittexts and one button.
When one edit text lose the focus, the library would validate just this particular view. 
When the button is clicked, the library would validate all edittexts using the current features.

**Comment by ragunathjawahar** [comment]
Yes, you can.

1. Use the `@Order` annotation on all your UI fields.
2. Set your `Validator`'s validation mode to `Mode.IMMEDIATE`.
3. Use a `TextWatcher` or an `OnFocusChangeListener` to validate your views by calling the `validator.validateTill(View)` or `validator.validateBefore(View)` methods to achieve the desired result.

**Comment by IvanSop** [comment]
Hi, I am trying to achieve, I believe the same thing, to validate single field that just lost focus, and this doesnt help me.
I have several inputs
Input1
Input2
Input3
If I annotate them in @Order (1,2,3) and set mode immediate.
If they are all invalid and user for example focuses Input3, then Input2, only Input1 will show as invalid
using validateTill(), but I need just input3 to show as invalid.
Basically, functionallity should be: you are focused on one input, and as it loses focus, its lit as invalid, no other fields are affected.
